L1Trigger/L1TGlobal/interface/AXOL1TLCondition.h

```diff
@@ -68,6 +73,9 @@ namespace l1t {
 
     /// pointer to uGt GlobalBoard, to be able to get the trigger objects
     const GlobalBoard* m_gtGTB;
+
+    ///axo score for possible score saving
+    mutable float m_savedscore;
```

Comment:
I would naively think that `mutable` should be avoided, unless truly necessary.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rp-mutable

Reply:
I set this as mutable deliberately: the score is calculated within the conditions class, and I did not have to recompute it more than necessary, so I set this internal variable to be fetched by the GlobalBoard. In order to be compatible with the inherited class structure and threading, using a mutable score was the best solution I found.